resources/wikia/modules/browserDetect.js

```diff
@@ -50,6 +50,14 @@ define('wikia.browserDetect', ['wikia.window'], function (win) {
 		return !!userAgent.match(/iPad.+OS.[6,7].\d.+like.Mac.OS.+Safari/i);
 	}
 
+	function isTouchScreen() {
```

Comment:
1) please add documentation
2) can you browse through code and replace ``Wikia.isTouchScreen()``` with that? You could also remove former library / function.